services/discourse-docs-snapcraft-io.yaml

```diff
@@ -0,0 +1,45 @@
+---
+
+kind: Service
+apiVersion: v1  # See https://bit.ly/2KdOtrZ
+metadata:
+  name: discourse-docs-snapcraft-io
+spec:
+  selector:
+    app: discourse-docs.snapcraft.io
+  ports:
+    - name: http
+      protocol: TCP
+      port: 80
+      targetPort: http
+
+---
+
+kind: Deployment
+apiVersion: extensions/v1beta1  # See https://bit.ly/2KdOtrZ
+metadata:
+  name: discourse-docs-snapcraft-io
+spec:
+  replicas: 5
+  template:
+    metadata:
+      labels:
+        app: discourse-docs.snapcraft.io
+    spec:
+      containers:
+        - name: discourse-docs-snapcraft-io
+          image: prod-comms.docker-registry.canonical.com/discourse-docs.snapcraft.io:${TAG_TO_DEPLOY}
+          ports:
+            - name: http
+              containerPort: 80
+          readinessProbe:
+            httpGet:
+              path: /_status/check
```

Comment:
It seems like this endpoint does not exist!

Reply:
We tried with `/` and got a gunicorn error.
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python3.6/dist-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.6/dist-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.6/dist-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.6/dist-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.6/dist-packages/talisker/gunicorn.py", line 205, in load_wsgiapp
    app = super(TaliskerApplication, self).load_wsgiapp()
  File "/usr/local/lib/python3.6/dist-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.6/dist-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
ModuleNotFoundError: No module named 'app'
```

Reply:
Sorry you're right, that was a bad image.

I've changed it to `export TAG_TO_DEPLOY=1536921430-286eac2` in the QA steps. That image seems to work okay:

``` bash
$ docker run -d -p 8999:80 prod-comms.docker-registry.canonical.com/discourse-docs.snapcraft.io:1536921430-286eac2
ea5014eed7db8a4e592f03cad0134b8c2ac05209e188056514a5770c041f575f
$ curl localhost:8999/_status/check
1536921430-286eac2
```